lib/glific/flows.ex

```diff
@@ -742,7 +743,7 @@ defmodule Glific.Flows do
                }),
              {:ok, _flow_revision} <-
                FlowRevision.create_flow_revision(%{
-                 definition: flow_revision["definition"],
+                 definition: clean_flow_with_template(flow_revision["definition"]),
```

Comment:
```suggestion
                 definition: clean_flow_for_templates(flow_revision["definition"]),
```

Also, is it for HSM or interactive, please add that name as well.

Reply:
made requested changes